<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChageUserAccountWidget</class>
 <widget class="QWidget" name="ChageUserAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChageUserAccountWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>90</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>100</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="rightMargin">
        <number>13</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>17</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>17</number>
         </property>
         <item>
          <widget class="QLabel" name="verifySrcphoneLbl">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>验证原绑定手机号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="telPhoneLbl">
           <property name="text">
            <string>152****0000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>31</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>34</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>34</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="oldVerifyEdt">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>黑体</family>
                    <pointsize>26</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="frame">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>验证码</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="oldVerifyBtn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>26</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>获取验证码</string>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
           <zorder></zorder>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>17</number>
           </property>
           <item>
            <widget class="QLabel" name="verifyDstPhoneLbl">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>绑定新手机号</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>12</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <property name="spacing">
                   <number>1</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>34</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>34</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lbl86">
                      <property name="font">
                       <font>
                        <family>黑体</family>
                        <pointsize>24</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>+86</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="newPhoneEdt">
                    <property name="font">
                     <font>
                      <family>黑体</family>
                      <pointsize>26</pointsize>
                     </font>
                    </property>
                    <property name="frame">
                     <bool>false</bool>
                    </property>
                    <property name="placeholderText">
                     <string>手机号</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>38</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <property name="spacing">
                     <number>1</number>
                    </property>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <item>
                       <widget class="QLabel" name="label_7">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>30</width>
                          <height>34</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>34</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="newVerifyEdt">
                        <property name="font">
                         <font>
                          <family>黑体</family>
                          <pointsize>24</pointsize>
                         </font>
                        </property>
                        <property name="frame">
                         <bool>false</bool>
                        </property>
                        <property name="placeholderText">
                         <string>验证码</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="newVerifyBtn">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="font">
                       <font>
                        <family>黑体</family>
                        <pointsize>24</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>获取验证码</string>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="cancelBtn">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>黑体</family>
                     <pointsize>24</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>取消</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="OKBtn">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>黑体</family>
                     <pointsize>24</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>确定</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
